tests/unit/experiments/providers/variant_sets/multi_variant_set_tests.py

```diff
@@ -0,0 +1,120 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import logging
+import collections
+import time
+import unittest
+
+from datetime import timedelta
+
+from baseplate._compat import range
+from baseplate.experiments.providers.variant_sets.multi_variant_set import MultiVariantSet
+
+logger = logging.getLogger(__name__)
+
+
+NUM_BUCKETS_DEF = 1000
+NUM_BUCKETS_ODD = 1037
+
+
+def generate_variant_config():
+    cfg = [
+        {"name": "variant_1", "size": 0.25},
+        {"name": "variant_2", "size": 0.25},
+        {"name": "variant_3", "size": 0.25},
+    ]
+
+    return cfg
+
+def create_multi_variant_set():
+    cfg = generate_variant_config()
+    return MultiVariantSet(cfg)
```

Comment:
IIRC you can pass in the number of buckets, you should probably pass it in here, maybe making it an argument to `create_multi_variant_set` with a default value of `NUM_BUCKETS_DEF` so you don't have to worry about keeping those in sync with the default values.

Reply:
fixed